torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -3065,6 +3065,7 @@ c10::intrusive_ptr<Work> ProcessGroupNCCL::pointToPoint(
 
   // Bump the logical operation counter regardless of whether this op is
   // coalesced or individual
+  seqCollective_++;
```

Comment:
seqCollective_ is not intended to be increments in `pointToPoint` operations.

Reply:
Thanks for the comment. Yes. Maybe we need to use `op_id_` for both collective and p2p execution trace dump.

Reply:
Do you need a unique identifier for each collective?

Reply:
@fduwjj Not only collective, but also p2p and coalesced op. This is used to matched with WorkNCCL in dumped Execution Trace.
For example (Surely, this is also a bug to be fixed): 
[https://github.com/pytorch/pytorch/blob/0c7856973b2c81edd5b65cadd349d3c6aff3ddb1/torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp#L2451](https://github.com/pytorch/pytorch/blob/0c7856973b2c81edd5b65cadd349d3c6aff3ddb1/torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp#L2451)

I have created another PR to fix this problem. [https://github.com/pytorch/pytorch/pull/134578](https://github.com/pytorch/pytorch/pull/134578)